velero-plugin-for-alibabacloud/volume_snapshotter.go

```diff
@@ -166,6 +167,7 @@ func (b *VolumeSnapshotter) CreateSnapshot(volumeID, volumeAZ string, tags map[s
 	req := ecs.CreateCreateSnapshotRequest()
 	req.DiskId = volumeID
 
+	tags["volumeAZ"] = volumeInfo.ZoneId
```

Comment:
Snapshots are not necessarily restored in the same zone that where they were created. Snapshots themselves are regionalized.

Reply:
there should be more thoughts. @AlbeeSo. any comments?